backend/src/main/java/com/yigongil/backend/application/StudyService.java

```diff
@@ -135,10 +137,34 @@ public List<StudyMemberResponse> findApplicantsOfStudy(Long studyId, Member mast
         study.validateMaster(master);
         List<StudyMember> studyMembers = studyMemberRepository.findAllByStudyIdAndRole(studyId, Role.APPLICANT);
 
-        return studyMembers.stream()
-                .map(StudyMember::getMember)
-                .map(StudyMemberResponse::from)
-                .toList();
+        return createStudyMemberResponses(studyMembers);
+    }
+
+    private List<StudyMemberResponse> createStudyMemberResponses(List<StudyMember> studyMembers) {
+        List<StudyMemberResponse> response = new ArrayList<>();
+        for (StudyMember studyMember : studyMembers) {
+            StudyMemberResponse studyMemberResponse = createStudyMemberResponse(studyMember);
+            response.add(studyMemberResponse);
+        }
+        return response;
```

Comment:
```suggestion
        return studyMembers.stream()
            .map(this::createStudyMemberResponse)
            .toList();
```
p5: 모던자바 expert 주드가 스트림을 안쓴다고??? 혹시 제가 생각못한 스트림 썼을 때 단점이 있다면 공유해주세요 🔥